README.md

```diff
@@ -119,6 +124,162 @@ resource provided above.
+    ```
+   + Open *id_rsa.pub* with either notepad or notepad++.
+
+   + Copy everything in that file and set up your key in your [github page](https://github.com/settings/keys).
+---
+  + Install [Postgresql](http://www.postgresql.org/)
+
+      (NOTE: Google Chrome will block these downloads for security reasons. Please download using Firefox or Edge)
+
+      Install postgres from [EnterpriseDB](https://www.enterprisedb.com/downloads/postgres-postgresql-downloads)
+
+      Set up your **username** and **password** defaults. You will not need to set up a specialized name or password when developing in Windows, but you will need to add these to the **database.yml** file later.
+
+      Install PGAdmin4 with postgres in the installer, or download manually from [PGAdmin for Windows](https://www.pgadmin.org/download/pgadmin-4-windows/)
+---
++ Set up the Database
+
+    Run PGAdmin4
+
+    +  If PGAdmin4 will not run, try the following:
+
+        Clear all Roaming Data:
+
+        ```
+        C:\Users\YourUserName\AppData\Roaming\pgAdmin
+        ```
+        Download [Oracle](https://www.oracle.com/technetwork/java/javase/downloads/index.html) Java and set up the [evironment variable path](https://www.addictivetips.com/windows-tips/set-path-environment-variables-in-windows-10/)
+
+        ```
+        C:\Program Files (x86)\Common Files\Oracle\Java\javapath
+        ```
+
+    Once PGAdmin4 is running, click on your servers, right click on *PostreSQL*, hover over *Create*, and then click *Database*
+
+    Type *shescoding_development* in the *Database* field and select the owner. I recommend using your general postgres user
+
++ Set up your local repo
+
+  + Open your Ubuntu for Windows Application
+
+  + Change your directory to your Windows location, such as Desktop
+
+    ```
+    $ cd /mnt/c/users/YourWindowsUserName/desktop
+    ```
+
+  + You can see the Ubuntu file directory if you wish, but it is not necessary unless you need to copy your SSH Keys for Github or other repositories
+
+    ```
+    C:\Users\YourWindowsUserName\AppData\Local\Packages\CanonicalGroupLimited.Ubuntu16.04onWindows_79rhkp1fndgsc\LocalState\rootfs\home\YourUbuntuUserName
+    ```
+  + Now clone the repository
+
+    ```
+    $ git clone https://github.com/shescoding/shescoding-dot-org.git
+    ```
+  + Add the upstream repository:
+
+    ```
+    $ git remote add upstream https://github.com/shescoding/shescoding-dot-org.git
+    ```
+  + Change the directory to your development environment
+
+  ```
+  $ cd shescoding-dot-org
+  ```
+  + Open your database.yml file and make the following changes
+    + Under **default: &default**
+
+    ```
+    host: localhost
+    ```
+
+    + Under **development:**
+
+    ```
+    username: YourPostGresUserName
+    password: YourPostGresPassword
+    ```
++ **With PGAdmin4 still running**, complete setup by following the steps [below](#mac,-ubuntu,-and-windows).
```

Comment:
`#mac-ubuntu-and-windows` is the correct link

Reply:
This has been fixed (strangely it worked on my end with the commas but not without. Not true with github)